test/ecto/changeset/belongs_to_test.exs

```diff
@@ -61,7 +62,7 @@ defmodule Ecto.Changeset.BelongsToTest do
 
   test "cast belongs_to with invalid params" do
     changeset = cast(%Author{}, %{"profile" => %{name: nil}}, :profile)
-    assert changeset.changes.profile.changes == %{}
+    assert changeset.changes.profile.changes == %{name: nil}
```

Comment:
this is seems to be the only non-obvious change in tests (apart from silencing warnings later in the PR)